exporter/exporter.go

```diff
@@ -1563,6 +1564,10 @@ func (e *Exporter) connectToRedis() (redis.Conn, error) {
 		uri = "redis://" + uri
 	}
 
+	if e.options.PasswordMap[uri] != "" {
+		options = append(options, redis.DialPassword(e.options.PasswordMap[uri]))
```

Comment:
This is not being hit by a test:
https://coveralls.io/builds/35963780/source?filename=exporter%2Fexporter.go#L1568

Can you make sure this is really being triggered by the test below?